src/Linkification/Embedding.coffee

```diff
@@ -418,4 +442,7 @@ Embedding =
           for item in data.items when item.id is uid
             return item.snippet.title
           'Not Found'
+      preview:
+        url: (uid) -> "https://img.youtube.com/vi/#{uid}/0.jpg"
+        height: 480
```

Comment:
Should be 360 (480 is the width).
Only posting again because the last comment got marked "outdated" although it wasn't.